packages/grafana-ui/src/components/Layout/Box/Box.tsx

```diff
@@ -268,6 +272,9 @@ const getStyles = (
       getResponsiveStyle(theme, boxShadow, (val) => ({
         boxShadow: theme.shadows[val],
       })),
+      getResponsiveStyle<ThemeSpacingTokens>(theme, gap, (val) => ({
```

Comment:
```suggestion
      getResponsiveStyle(theme, gap, (val) => ({
```

I think we actually don't need this and it is inferred